modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContext.java

```diff
@@ -258,6 +259,29 @@ public ObjectLayout getObjectLayout() throws PortalException {
 		}
 	}
 
+	public ObjectLayoutBox getObjectLayoutBoxCategorization()
+		throws PortalException {
+
```

Comment:
Hi @rafaprax what do you thing of adding an if to check if the ObjectDefinition StorageType is "default" here? If I do it, we don't need to pass by the foreach in this case.

Reply:
it's ok to me.